newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -142,6 +158,21 @@ export default class SearchPanel extends PureComponent<Props, State> {
                 checked={!this.state.matchCase}
                 onCheck={(e, checked) => this.setState({ matchCase: !checked })}
               />
+              <p>{'Filter by:'}</p>
```

Comment:
Use `<Trans>` to make sure this string can be translated :) 

`<p><Trans>Filter by</Trans></p>`